doc/src/manual/variables.md

```diff
@@ -104,7 +104,7 @@ Operators like `+` are also valid identifiers, but are parsed specially. In some
 can be used just like variables; for example `(+)` refers to the addition function, and `(+) = f`
 will reassign it. Most of the Unicode infix operators (in category Sm), such as `⊕`, are parsed
 as infix operators and are available for user-defined methods (e.g. you can use `const ⊗ = kron`
-to define `⊗` as an infix Kronecker product).  Operators can also be suffixed with modifying marks,
+to define `⊗` as an infix [Kronecker product](https://en.wikipedia.org/wiki/Kronecker_product)).  Operators can also be suffixed with modifying marks,
```

Comment:
You should break the line (also below) after 80 symbols.

Reply:
Hello,
Thanks for the suggestion. 
Please let me know which line need to be break as per 80 symbols. Is the line starting with "Operators can also be suffixed with modifying marks," should have line break.

Reply:
All the ones that are longer than 80 chars.